### PR TITLE
add extension to relative imports to make the code work directly in a browser

### PR DESCRIPTION
**Pull request by JakeChampion**
relative imports require the full path including the extension because browsers use the import name to make a network request and they do not implicitly add a .js file extension if one is missing

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import Video from './src/js/video.js';
+import Video from './src/js/video.js.js';
```
Comment (JakeChampion) [inline main.js]:
```suggestion
import Video from './src/js/video.js';
```

File: test/helpers/get-rendition.test.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 /* eslint-env mocha */
 /* global proclaim */
 
-import getRendition from './../../src/js/helpers/get-rendition';
-import { mediaApiResponse1 } from '../fixtures/media-api-1.js';
+import getRendition from './../../src/js/helpers/get-rendition.js';
+import { mediaApiResponse1 } from '../fixtures/media-api-1.js.js';
```
Comment (JakeChampion) [inline test/helpers/get-rendition.test.js]:
```suggestion
import { mediaApiResponse1 } from '../fixtures/media-api-1.js';
```

File: test/video.test.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 /* eslint-env mocha */
 /* global proclaim sinon */
 
-import Video from './../src/js/video';
-import {mediaApiResponse1} from './fixtures/media-api-1.js';
-import {mediaApiResponse2} from './fixtures/media-api-2.js';
+import Video from './../src/js/video.js';
+import {mediaApiResponse1} from './fixtures/media-api-1.js.js';
```
Comment (JakeChampion) [inline test/video.test.js]:
```suggestion
import {mediaApiResponse1} from './fixtures/media-api-1.js';
```

File: test/video.test.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 /* eslint-env mocha */
 /* global proclaim sinon */
 
-import Video from './../src/js/video';
+import Video from './../src/js/video.js';
 import {mediaApiResponse1} from './fixtures/media-api-1.js';
-import {mediaApiResponse2} from './fixtures/media-api-2.js';
+import {mediaApiResponse2} from './fixtures/media-api-2.js.js';
```
Comment (JakeChampion) [inline test/video.test.js]:
```suggestion
import {mediaApiResponse2} from './fixtures/media-api-2.js';
```

**Comment by origamiserviceuser** [comment]
o-video bundle size difference from 6.0.10 to 81a92851714b70ba04bc54c23c4e1992447b98c8
No significant bundle size differences found.

**Review by notlee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v6.0.11 :tada:

**Pull request merged**